src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs

```diff
@@ -117,15 +119,32 @@ private void SortAndRemoveUnusedImports(ITextBuffer subjectBuffer, CancellationT
             var document = subjectBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
             if (document != null)
             {
-                var newDocument = document.GetLanguageService<IRemoveUnnecessaryImportsService>().RemoveUnnecessaryImportsAsync(document, cancellationToken).WaitAndGetResult(cancellationToken);
+                var addMissingImportsService = document.GetLanguageService<IAddMissingImportsFeatureService>();
```

Comment:
I would add this in code clean up rather than organize document (or sort and remove unnecessary imports command). especially if it only works if it thinks it is safe (no ambiguity)